schedule/alp/alp_yast.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: alp_yast
+description: >
+    Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
```

Comment:
move this line under the real description is better, WDYT?

Reply:
Deleted it as other schedule files.